types/bpmn-moddle/tslint.json

```diff
@@ -0,0 +1,79 @@
+{
```

Comment:
For a new definition you shouldn't need to disable all these rules.

Reply:
No worries. I removed all the special rules and specifically only extended dt.json in the latest commit.